packages/net/tcp_connection.pony

```diff
@@ -632,9 +632,7 @@ actor TCPConnection
             // The connection failed, unsubscribe the event and close.
             @pony_asio_event_unsubscribe(event)
             @pony_os_socket_close(fd)
-            if _connect_count == 0 then
-              _notify.connect_failed(this)
-            end
+            _notify_connecting()
```

Comment:
this is not the failure cause, but i dont see a reason to not call _notify_connecting()

@kulibali why did you change it previously?

Reply:
that said, this is not causing the bug so it could be left if there is a reason.